HoloPipelines/pipelineController.py

```diff
@@ -45,8 +45,8 @@ def main():
         os.mkdir("medicalScans/nifti")
     if not os.path.exists("output"):
         os.mkdir("output")
-        os.mkdir("output/OBJ")
-        os.mkdir("output/GLB")
+        os.mkdir("output/obj")
+        os.mkdir("output/glb")
```

Comment:
It's does not look right for paths to be hardcoded here when your other components take reference elsewhere. It would be better if you use the same reference everywhere it is needed.

Reply:
Changed to compCommonPath